src/common/CMakeLists.txt

```diff
@@ -93,5 +93,6 @@ if (BUILD_MATLAB)
   INSTALL(DIRECTORY +sirf DESTINATION "${MATLAB_DEST}")
   file(GLOB MatlabFiles "${CMAKE_CURRENT_LIST_DIR}/*.m")
   INSTALL(FILES ${MatlabFiles} DESTINATION "${MATLAB_DEST}")
+  configure_file("${CMAKE_CURRENT_SOURCE_DIR}/+sirf/+SIRF/+Utilities/examples_data_path.m" ${MATLAB_DEST}/+SIRF/+Utilities/examples_data_path.m)
```

Comment:
I'd file more comfortable with having a `.m.in` file. Or better: we create a `version.m.in` , and get the info from there